upx-recovery-tool.py

```diff
@@ -0,0 +1,289 @@
+#! /usr/bin/python3
+
+import os
+import re
+import mmap
+import magic
```

Comment:
Afaik, magic is a non-standard module and therefore should be added to requirements.txt